okhttp/src/main/java/com/squareup/okhttp/MediaType.java

```diff
@@ -31,6 +31,64 @@
   private static final Pattern PARAMETER = Pattern.compile(
       ";\\s*(?:" + TOKEN + "=(?:" + TOKEN + "|" + QUOTED + "))?");
 
+  /**
+   * A {@link MediaType} constant representing the wildcard media type.
+   */
+  public static final MediaType WILDCARD = new MediaType("*/*", "*", "*", null);
+  /**
+   * A {@link MediaType} constant representing the "application/xml" media type.
+   */
+  public static final MediaType APPLICATION_XML =
+                  new MediaType("application/xml", "application", "xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/atom+xml" media type.
+   */
+  public static final MediaType APPLICATION_ATOM_XML =
+                  new MediaType("application/atom+xml", "application", "atom+xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/xhtml+xml" media type.
+   */
+  public static final MediaType APPLICATION_XHTML_XML =
+                  new MediaType("application/xhtml+xml", "application", "xhtml+xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/svg+xml" media type.
+   */
+  public static final MediaType APPLICATION_SVG_XML =
+                  new MediaType("application/svg+xml", "application", "svg+xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/json" media type.
+   */
+  public static final MediaType APPLICATION_JSON =
+                  new MediaType("application/json", "application", "json", null);
```

Comment:
(I guess you'd also have to change the first argument, to be like `application/json; charset=utf-8`)